docs/source/apps/projsync.rst

```diff
@@ -149,7 +149,7 @@ Examples
     Bugs
     ****
 
-    A list of know bugs can be found at https://github.com/OSGeo/PROJ/issues
+    A list of known bugs can be found at https://github.com/OSGeo/PROJ/issues
```

Comment:
Shouldn't this also include the `comman_man.rst` file?

Reply:
no, because of the extra addition, one line later, about reporting resource related issues to PROJ-data github